performance-timeline/po-resource.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>PerformanceObservers: resource</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="performanceobservers.js"></script>
+<h1>PerformanceObservers: resource</h1>
+<p>
+New resources will <a href="https://w3c.github.io/performance-timeline/#dfn-queue-a-performanceentry">queue a PerformanceEntry</a>.
+</p>
+<div id="log"></div>
+<script>
+  async_test(function (t) {
+    var stored_entries = [];
+    var img_location = document.location.origin
+       + "/performance-timeline/resources/square.png?random=";
```

Comment:
This makes it impossible to run via http://w3c-test.org/submissions/2884/... Why not just a relative "resources/square.png"?